src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Interop.crypt32.cs

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using CryptographicException = System.Security.Cryptography.CryptographicException;
+
+using SafeX509ChainHandle = Microsoft.Win32.SafeHandles.SafeX509ChainHandle;
+using X509KeyUsageFlags = System.Security.Cryptography.X509Certificates.X509KeyUsageFlags;
+
+namespace Internal.Cryptography.Pal.Native
+{
+    internal static partial class Interop
+    {
+        public static class crypt32
```

Comment:
I assume, as with some of the other crypto libs, that we plan to but are deferring rationalizing this with how we're doing interop files elsewhere in corefx?